lib/utilities.js

```diff
@@ -41,13 +42,30 @@ async function getDeviceName (udid, socket) {
   }
 }
 
+/**
+ * @typedef {Object} DeviceTime
+ *
+ * @property {number} timestamp Unix timestamp in seconds since 1970-01-01T00:00:00Z
+ * @property {number} utcOffset The difference in seconds between the UTC time and the local device time.
+ * Can be negative.
+ */
+
+/**
+ * Retrieves the local time from the device under test
+ *
+ * @param {string} udid Device UDID
+ * @param {Object} socket
+ * @returns {DeviceTime}
+ */
 async function getDeviceTime (udid, socket) {
   const lockdown = await startLockdownSession(udid, socket);
   try {
-    const epochValue = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_TIME);
-    const date = new Date(0); // The 0 there is the key, which sets the date to the epoch
-    date.setUTCSeconds(epochValue);
-    return date;
+    const timestamp = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_TIME);
+    const utcOffset = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_UTC_OFFSET);
```

Comment:
Have you tested this? I think this returns a double

Reply:
the client will be using moment.js. I don't want to perform double transformation

Reply:
this is fine